internal/juju/kubernetesClouds.go

```diff
@@ -20,6 +20,16 @@ import (
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 )
 
+const (
+	// WorkloadStorageKey is the model config attribute used to specify
+	// the storage class for provisioning workload storage.
+	WorkloadStorageKey = "workload-storage"
```

Comment:
2 questions:
- do these two consts need to be exported (i do not think they do)
- what is the difference between `workload` and `operator` storage?